packages/block-editor/src/components/link-control/index.js

```diff
@@ -0,0 +1,153 @@
+	const [ inputValue, setInputValue ] = useState( '' );
+	const [ isEditingLink, setIsEditingLink ] = useState( true );
+
+	// Handlers
+	const onInputChange = ( value = '' ) => {
+		setInputValue( value );
+	};
+
+	const closeLinkUI = () => {
+		resetInput();
+	};
+
+	const resetInput = useCallback( () => {
+		setInputValue( '' );
+	} );
+
+	const onLinkSelect = ( event, suggestion ) => {
+		event.preventDefault();
+		event.stopPropagation();
+
+		setIsEditingLink( false );
+
+		if ( isFunction( onLinkChange ) ) {
+			onLinkChange( suggestion );
+		}
+	};
+
+	const onStartEditing = () => {
+		setIsEditingLink( true );
+	};
+
+	const handleURLSearch = async ( value ) => {
+		return [ {
+			id: 1,
+			title: value,
+			type: 'URL',
+			url: prependHTTP( value ),
+		} ];
+	};
+
+	// Effects
+	const getSearchHandler = useCallback( ( value ) => {
+		return ( isURL( value ) || value.includes( 'www.' ) ) ? handleURLSearch( value ) : fetchSearchSuggestions( value );
+	}, [ handleURLSearch, fetchSearchSuggestions ] );
+
+	// Render Components
+	const renderSearchResults = ( { suggestionsListProps, buildSuggestionItemProps, suggestions, selectedSuggestion } ) => {
+		/* eslint-disable react/jsx-key */
+		return (
+			<div className="block-editor-link-control__search-results-wrapper">
+				<div { ...suggestionsListProps } className="block-editor-link-control__search-results">
+					{ suggestions.map( ( suggestion, index ) => (
+						<LinkControlSearchItem
+							key={ `${ suggestion.id }-${ index }` }
+							itemProps={ buildSuggestionItemProps( suggestion, index ) }
+							suggestion={ suggestion }
+							onClick={ partialRight( onLinkSelect, suggestion ) }
+							isSelected={ index === selectedSuggestion }
+							isURL={ suggestion.type.toLowerCase() === 'url' }
+							searchTerm={ inputValue }
+						/>
+					) ) }
+				</div>
+			</div>
+		);
+		/* eslint-enable react/jsx-key */
+	};
+
+	return (
+		<Popover
+			className="block-editor-link-control"
+			onClose={ closeLinkUI }
+			position="bottom center"
+			focusOnMount="firstElement"
+		>
+			<div className="block-editor-link-control__popover-inner">
+				<div className="block-editor-link-control__search">
+
+					{ ! isEditingLink && (
+						<div
+							className={ classnames( 'block-editor-link-control__search-item', {
+								'is-current': true,
+							} ) }
+						>
+							<span className="block-editor-link-control__search-item-header">
+
+								<ExternalLink
+									className="block-editor-link-control__search-item-title"
+									href={ currentLink.url }
+								>
+									{ currentLink.title }
+								</ExternalLink>
+								<span className="block-editor-link-control__search-item-info">{ currentLink.info || filterURLForDisplay( safeDecodeURI( currentLink.url ) ) || '' }</span>
+							</span>
+
+							<Button isDefault onClick={ onStartEditing } className="block-editor-link-control__search-item-action block-editor-link-control__search-item-action--edit">
+								Change
```

Comment:
```suggestion
								{ __( 'Change' ) }
```

Reply:
Thanks. Fixed this manually as had to include the `__()` function utility itself.